src/components/elements/off-canvas-menu.js

```diff
@@ -365,31 +365,44 @@ export const OffCanvasMenuWrapper = (props) => {
                             </div>
                             <span>{localize('Help centre')}</span>
                         </StyledLink>
-                        {/* TODO: add this when blog is ready */}
-                        {/* <StyledLink to="/blog/" onClick={handleArrowClick}>
-                            {localize('Blog')}
-                        </StyledLink> */}
-                        <StyledLink to="/payment-methods/" onClick={handleArrowClick}>
+                        <StyledLink
+                            to=""
```

Comment:
Does `to` here have special meaning or it's just an empty link?

Reply:
It's a requirement for the `localizedlink` to have a `to` prop. However, the `is_blog_link` / `is_community_link` already covers the url requirement.